resources/static/images/metamask.svg

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
what's the copyright status on this?

Reply:
It's licensed under MIT, so should be fine